includes/class-comment.php

```diff
@@ -70,7 +70,7 @@ public static function register_comment_types() {
 				'singular'    => __( 'Repost', 'webmention' ),
 				'description' => __( 'A repost on the indieweb is a post that is purely a 100% re-publication of another (typically someone else\'s) post.', 'webmention' ),
 				'icon'        => '♻️',
-				'class'       => 'u-repost-of',
+				'class'       => 'p-repost',
```

Comment:
wasn't `*-of` or `*-to` the preferred way? https://indieweb.org/reply#How_To

Reply:
For marking it up for publishing, not showing other people's reposts on your post.

Reply:
ok, thanks for clarification!